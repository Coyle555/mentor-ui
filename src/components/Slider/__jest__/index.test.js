import React from 'react';
import {
	render,
	cleanup,
	fireEvent,
	getByTestId,
} from '@testing-library/react';

import {
	Slider,
	calcHandlePos,
	getHandleStyle,
	getValueWidthStyle,
	normalizer,
	onMove,
	KNOB_SIZE,
	BORDER_OFFSET,
} from 'components/Slider';

beforeEach(cleanup);

describe('Slider component', () => {
	it('Should default render', () => {
		const { container } = render(<Slider />);

		expect(container).toMatchSnapshot();
	});

	it('Should match snapshot with className prop', () => {
		const className = "RidelsMPA";
		const { container } = render(
			<Slider
				className={className}
			/>
		);

		expect(container).toMatchSnapshot();
	});
});
