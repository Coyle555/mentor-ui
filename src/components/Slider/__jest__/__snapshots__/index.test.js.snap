// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slider component Should default render 1`] = `
<div>
  <div
    class="APMSlider"
  >
    <div
      class="APMSlider-value"
    />
    <div
      class="APMSlider-handle"
      style="height: 38px; width: 38px;"
    />
  </div>
</div>
`;

exports[`Slider component Should match snapshot with className prop 1`] = `
<div>
  <div
    class="APMSlider RidelsMPA"
  >
    <div
      class=" APMSlider-value RidelsMPA-value"
    />
    <div
      class=" APMSlider-handle RidelsMPA-handle"
      style="height: 38px; width: 38px;"
    />
  </div>
</div>
`;
